File: practicas-java/numeromayor/NumeroMayor.java
/*
 Pedir dos números y decir cuál es el mayor
 */
package numeromayor;
import java.util.Scanner;

public class NumeroMayor {

    public static void main(String[] args) {
        Scanner entrada=new Scanner(System.in);
        
        int number1,number2;
        
        System.out.println("Ingrese el primer número");
        number1=entrada.nextInt();
        
        System.out.println("Ingrese el segundo número");
        number2=entrada.nextInt();
        
        if(number1 > number2){
            System.out.println("El 1er número es mayor");
        }else{
            System.out.println("el 2do numero es mayor");
        }
    }
    
}

File: practicas-en-c/numeroMayor/numeroMayor.c
//Pedir dos números y decir cual es el mayor
#include<stdio.h>
#include<stdlib.h>

int main(){
	int numero1,numero2;
	
	printf("Ingrese el 1er numero: ");
	scanf("%d",&numero1);
	
	printf("Ingrese el 2do numero: ");
	scanf("%d",&numero2);
	
	if(numero1>numero2){
		printf("%d es mayor a %d \n",numero1,numero2);
	}else if(numero2>numero1){
		printf("%d es mayor a %d \n",numero2,numero1);
	}else{
		printf("Ambos son iguales \n");
	} 
	
	system("PAUSE");
	return 0;
}
